src/mds/Migrator.cc

```diff
@@ -169,17 +169,6 @@ void Migrator::dispatch(Message *m)
   }
 }
 
-
-class C_MDC_EmptyImport : public MigratorContext {
-  CDir *dir;
-public:
-  C_MDC_EmptyImport(Migrator *m, CDir *d) : MigratorContext(m), dir(d) {}
-  void finish(int r) override {
-    mig->export_empty_import(dir);
-  }
-};
-
-
```

Comment:
Dead code? Separate commit and PR please. (leaving this in here will mess up backports.)